mappymatch/constructs/trace.py

```diff
@@ -46,6 +46,17 @@ def __add__(self, other: Trace) -> Trace:
     def __len__(self):
         return len(self._frame)
 
+    def __str__(self):
+        output_lines = [
+            "Mappymatch Trace object",
+            f'coords_list: {self.coords_list if hasattr(self, "coords_list") else None}',
```

Comment:
I think the coordinate list is actually defined as `coords` [here](https://github.com/NREL/mappymatch/blob/9347f6d7030e9c556655cdee05550ccde76b3465/mappymatch/constructs/trace.py#L54).